Tests/TuistGeneratorTests/Linter/TargetLinterTests.swift

```diff
@@ -274,6 +274,42 @@ final class TargetLinterTests: TuistUnitTestCase {
         }
     }
 
+    func test_lint_when_visionos_deployment_target_is_invalid_for_iPad_design() {
```

Comment:
The name here does not match what we're checking for.  I would expect `is_invalid` to check that we got linting errors, but we're using `XCTDoesNotContainLintingIssue` to ensure we did not get them

Reply:
woopsie :)

Reply:
lmk your thoughts